.github/workflows/tests.yml

```diff
@@ -310,11 +312,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # test lower and upper bounds
-        python-version: ['3.7', '3.10']
-        conda-version: [canary]
+        # test lower version (w/ stable conda) and upper version (w/ canary conda)
+        python-version: ['3.7']
+        conda-version: [release]
         test-type: [serial, parallel]
-      max-parallel: 10
+        include:
+          - python-version: '3.10'
+            conda-version: canary
+            test-type: serial
+          - python-version: '3.10'
+            conda-version: canary
+            test-type: parallel
```

Comment:
Do we want to allow these to fail?

Reply:
no? we should always be ready to release?